engine/samples/events/page.md

```diff
@@ -0,0 +1,28 @@
+# Events {#examples-engine-events}
+
+@brief Using the @ref cubos::core::ecs::EventReader and @ref cubos::core::ecs::EventWriter
```

Comment:
```suggestion
@brief Using the @ref cubos::core::ecs::EventReader "EventReader" and @ref cubos::core::ecs::EventWriter "EventWriter".
```

This way the namespace is hidden while still keeping the reference, becomes easier to read. :relieved: